input/2D/relaxation/distributed/poly_mps_ic_onestep_parallel.i

```diff
@@ -0,0 +1,336 @@
+[Mesh]
+ [gmg]
+  type = DistributedRectilinearMeshGenerator
+  dim = 2
+  nx = 125
+  ny = 125
+  xmin = 0
+  xmax = 20000
+  ymin = 0
+  ymax = 20000
+  partition = square
+  []
+  parallel_type = distributed
+[]
+
+[GlobalParams]
+  op_num = 12
+  var_name_base = etam
+  int_width = 480
+[]
+
+[UserObjects]
+  [./voronoi]
+    type = PolycrystalVoronoi
+    rand_seed = 1258
+    file_name = grains12.txt
+  [../]
+  [./grain_tracker]
+    type = GrainTracker
+    threshold = 0.2
+    connecting_threshold = 0.08
+  [../]
+[]
+
+[Variables]
+  [./PolycrystalVariables]
+  [../]
+[]
+
+[AuxVariables]
+  [./bubble]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./bnds]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./time]
+  [../]
+  # For use of GrainTracker
+  [./unique_grains]
```

Comment:
Do we have to define all these GrainTracker variables manually?

Reply:
Removed some of the variables that are not needed.